package.json

```diff
@@ -7,7 +7,7 @@
     "node": "^12.18"
   },
   "scripts": {
-    "build": "./node_modules/.bin/babel src -d ./build/server",
+    "build": "./node_modules/.bin/babel src -d ./build/server  && ./node_modules/.bin/babel config -d build/config",
```

Comment:
This change adds build/config/config.js to the transpiled server file. We needed this folder also in addition to src. Otherwise, the server wasn't able to read the db config.